bin/www

```diff
@@ -22,11 +22,11 @@ app.set('port', port);
 
 var server = http.createServer(app);
 
-models.sequelize.sync().then(function () {
-  server.listen(port);
-  server.on('error', onError);
-  server.on('listening', onListening);
-});
+// models.sequelize.sync().then(function () {
```

Comment:
Thanks I removed it!  I'm pretty terrified to break anything about how sequelize is working (which leads me to doubt that I no longer need something that was in place when it was working).  But I'm pretty confident I don't need this sync since my data is not changing at this point.